### PR TITLE
Do not reuse pkgdir across architectures

### PR DESCRIPTION
**Pull request by smarterclayton**
Also, remove final vestiges of atomic-enterprise naming

### CODE REVIEW

**Comment by smarterclayton** [comment]
[test]

File: Makefile [hunk 1]
```diff
@@ -249,32 +249,29 @@ install-travis:
 # Build RPMs only for the Linux AMD64 target
 #
 # Args:
-#   BUILD_TESTS: whether or not to build a test RPM, off by default
 #
 # Example:
 #   make build-rpms
 build-rpms:
-	BUILD_TESTS=$(BUILD_TESTS) OS_ONLY_BUILD_PLATFORMS='linux/amd64' hack/build-rpm-release.sh
```
Comment (stevekuznetsov) [inline Makefile]: Why?

Comment (smarterclayton) [inline Makefile]: We don't need it anymore, it doesn't cost us anything

Comment (stevekuznetsov) [inline Makefile]: That's amazing. Should also remove it from `origin.spec` then, it was only there to support us

**Comment by smarterclayton** [comment]
OK.

**Comment by smarterclayton** [comment]
[test]

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**